sklearn/cross_decomposition/pls_.py

```diff
@@ -390,6 +391,12 @@ def transform(self, X, Y=None, copy=True):
         -------
         x_scores if Y is not given, (x_scores, y_scores) otherwise.
         """
+        # XXX : don't know what to do with Y
```

Comment:
Sorry I don't follow this comment? What you are doing below is the right thing. Correct?